topics/icrc_34_get_global_delegation.md

```diff
@@ -0,0 +1,170 @@
+    * [Example RPC Request Permission](#example-rpc-request-permission)
+  * [Request](#request)
+    * [Example RPC Request](#example-rpc-request)
+    * [Response](#response)
+    * [Example RPC Response](#example-rpc-response)
+  * [Message Processing](#message-processing)
+  * [Errors](#errors)
+<!-- TOC -->
+
+## Summary
+
+When a relying party wants to authenticate as a user, it uses
+a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ecdsa), and
+below `icrc34_get_global_delegation` method to obtain a delegation chain that allows the session key to sign for the
+user's global identity. The obtained delegation chain **MUST** be restricted by the signer to canister targets that
+trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) standard.
+
+- Due to the canister target restriction, only the relying party its own canisters can be called on behalf of the user.
+- A new delegation chain must be obtained for every new canister created by the relying party.
+- The identity is the same across all relying parties, the user is not anonymous between different relying parties.
+
+## Method
+
+**Name and Scope:** `icrc34_get_global_delegation`
+
+**Prerequisite:** Active session with granted permission scope `icrc34_get_global_delegation` or `*`.
+
+## Scope (according to the [ICRC-25 standard](./icrc_25_signer_interaction_standard.md))
+
+**Scope:** `icrc34_get_global_delegation`
+
+**Required Properties:**
+
+- `targets` (`text` array): A list of target canister ids (textual representation) the scope is restricted to.
+
+### Example RPC Request Permission
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "icrc25_request_permissions",
+  "params": {
+    "scopes": [
+      {
+        "method": "icrc34_get_global_delegation",
+        "targets": [
+          "xhy27-fqaaa-aaaao-a2hlq-cai"
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Request
+
+**`principal` (`text`):** Principal that the global delegation is requested for.  
+**`publicKey` (`blob`):** Public key that receives the global delegation as described in
+the [IC interface specification, signatures section](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures).  
+**`targets` (`text` array):** A list of target canister ids (textual representation) the scope is restricted to.  
+**`maxTimeToLive` (`text` array):** (Optional) Expiration of the delegation in nanoseconds, signer can still choose to
+return a delegation with a shorter expiration.
+
+### Example RPC Request
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "icrc34_get_global_delegation",
+  "params": {
+    "principal": "gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-jae",
+    "publicKey": "MDwwDAYKKwYBBAGDuEMBAgMsAAoAAAAAAGAAJwEB9YN/ErQ8yN+14qewhrU0Hm2rZZ77SrydLsSMRYHoNxM=",
+    "targets": [
+      "xhy27-fqaaa-aaaao-a2hlq-cai"
+    ],
+    "maxTimeToLive": "28800000000000"
+  }
+}
+```
+
+### Response
```

Comment:
Similarly to the session delegation case below: The RP expects to receive a delegation on a public key matching the principal. However, to use that delegation the RP needs to know the public key as well. So it needs to be transferred here in the response too.